packages/forma-36-react-components/package.json

```diff
@@ -97,7 +97,7 @@
     "storybook-prop-types-addon": "1.0.2",
     "style-loader": "^0.23.1",
     "travis-deploy-once": "^4.4.1",
-    "typescript": "3.7.5",
+    "typescript": "3.9.7",
```

Comment:
😍

Reply:
I tried to update to v4 but I got too many errors so let's keep it for the next time 😄